go/host/enclave/guardian.go

```diff
@@ -632,6 +624,49 @@ func (g *Guardian) periodicRollupProduction() {
 	}
 }
 
+func (g *Guardian) periodicBundleSubmission() {
+	defer g.logger.Info("Stopping bundle submission")
+
+	// check rollup every l1 block time
+
+	interval := g.rollupInterval
+	if interval == 0 {
+		interval = g.blockTime
+	}
+
+	bundleSubmissionTicker := time.NewTicker(interval)
+
+	for {
+		select {
+		case <-bundleSubmissionTicker.C:
+			from, to, err := g.sl.L1Publisher().GetBundleRangeFromManagementContract()
```

Comment:
what are the odds of races here?
Something submitted, and not included yet, and then we resubmit?
How would the MC handle that?

(Btw, we have the same issue for rollups)